exercises/practice/dot-dsl/.meta/config.json

```diff
@@ -1,6 +1,47 @@
 {
   "blurb": "Write a Domain Specific Language similar to the Graphviz dot language",
   "authors": [],
```

Comment:
Another empty one, added in https://github.com/exercism/elixir/commit/cfb5f137f7d10f474a8870497ecf6678c0487c8f